_63C/digitalwallet.md

```diff
@@ -0,0 +1,14 @@
+
+In most distributed scenarios that digital identity wallets are applied to, the RP's trust in the attributes tends to come from the ultimate sources of the attributes, not from the wallet itself. This is in contrast to the IdP model presented in the Guidelines, where the RP's trust in the attributes stems from its trust in the IdP asserting those attributes in a federation transaction. From the perspective of the Guidelines, the IdP is the source of the attributes. The IAL and AAL for a given federation transaction are applied to the entire account being presented, with any attributes asserted as being a part of that account.
+
+Furthermore, while the IdP model in the Guidelines does allow for presentation of attributes through an identity API in addition to being present in the assertion itself (see [Q-C8](#q-c8) for details), even in these cases, the identity API is assumed to be under the control of the IdP and inherits its trust from the IdP. Advanced use cases involving independent attribute providers, attribute metadata (including provenance), and distributed attribute models are outside the scope of the Guidelines.
+
+That said, it is technically possible for a digital identity wallet to act as an IdP. While OpenID Connect and SAML are the most common federated identity protocols in use today, the Guidelines allow any protocol that meets the requirements outlined in the Guidelines to act as a federation protocol. Similarly, while a web-hosted server is the most common deployment for an IdP, the Guidelines do not assume any particular deployment model for IdPs. Given that the definition of IdP does not assume a publicly deployed web service, a digital identity wallet could serve the function of an IdP as long as it fulfills all the requirements laid out in the Guidelines. Namely, the wallet would need to create a signed assertion (which still needs to meet all the general assertion requirements) and somehow transfer that assertion to the RP over a network using either a front-channel or back-channel presentation mechanism. The RP in turn would need to be able to validate the assertion's signature and contents, as well as determine that the IdP is a trustworthy party for generating the assertion. There are numerous methods for establishing an IdP-RP trust relationship discussed in the Guidelines, and any of these could be applicable to trusting assertions from a digital identity wallet style application. However, these kinds of trust relationships are uncommon in the kinds of networks that digital identity wallets are designed for, with each user carrying their own independent wallet.
+
+In the end, while it's technically feasible for a wallet to act as an IdP, it would need to truly act in the role of an IdP including having the RP's connecting to the wallet trust the wallet's ability to present assertions and attributes.
```

Comment:
```suggestion
In the end, while it's technically possible for a wallet to act as an IdP, to be useful in practice it would need to truly act in that role. This includes the need for RPs to establish trust for assertions and attributes coming from the wallet.
```

Reply:
```suggestion
In the end, while it's technically possible for a wallet to act as an IdP, to be useful in practice it would need to truly act in that role. This includes the need for RPs to establish trust for assertions and attributes coming from the wallet as well as the wallet's ability to present these assertions and attributes to the RP.
```

Reply:
Didn't want to lose the presentation aspect to this, but the rest is fine. Incorporating both together, please take a look.